src/modules/ekf2/ekf2_params.c

```diff
@@ -1203,15 +1203,27 @@ PARAM_DEFINE_FLOAT(EKF2_PCOEF_XN, 0.0f);
 
 /**
  * Pressure position error coefficient for the Y axis.
```

Comment:
The title description should also be different between the 2 parameters, so e.g. `Pressure position error coefficient for the positive Y axis.`

Reply:
Fixed